src/lib/utils.py

```diff
@@ -19,6 +19,7 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 import datetime as dt
+import pytz
```

Comment:
https://blog.ganssle.io/articles/2018/03/pytz-fastest-footgun.html 

There is now a zoneinfo library in Python that can be used instead:

https://peps.python.org/pep-0615/
https://docs.python.org/3.9/library/zoneinfo.html

Let's use that rather than depending on pytz?

Reply:
Will try to look into it, thanks